haskell/private/pkgdb_to_bzl.py

```diff
@@ -25,7 +25,12 @@ def unfold_fields(content):
     """Unfold fields that were split over multiple lines.
 
     Returns:
-        A list of field name/value pairs.
+        A list of string, each string containing multiples lines
```

Comment:
Fewer words is better than more words. Here's a suggestion:

> A list of strings. Each string represents one field (a name/value pair separated by a colon).

Nothing else is particularly relevant to the user. Think of it as the `lines` function in Haskell (which likewise doesn't fiddle with whitespace, hence gets to likewise keep the documentation simple).

Reply:
I do not agree with your suggestion because the function is technically more generic than that and does not care at all about colon or name/value pair.

Suggestion applied.